File: PHP Word Examples/TablesApiDemo/Rows/getTableRowFormat.php
<?php

use Spire\Cloud\Word\Sdk\Client\Api\TablesApi;
use Spire\Cloud\Word\Sdk\Client\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId,$appKey,$baseUrl);
$apiInstance = new TablesApi($configuration);

$fileName = "GetTableRowFormat_1.docx";
$table_path = 'Section/0/Body/0/Table/0';
$index = 0;
$folder="Tables";
$password =null;
$storage = null;
$result = $apiInstance->getTableRowFormat($fileName, $table_path,$index,$folder,$storage,$password);

File: Python Word Examples/TablesApiDemo/Rows/getTableRowFormat.py
import spirecloudword
from spirecloudword.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey,baseUrl)
api= spirecloudword.api.TablesApi(configuration)

file_name = 'Template.docx'
table_path = 'Section/0/Body/0/Table/6'
index = 2
result =api.get_table_row_format(file_name, table_path, index, folder="input")